### DISCUSSION TITLE
Predict_survival_function returns implementation error.

### DISCUSSION THREAD

**Discussion by dlee07** [CamDavidsonPilon/lifelines#1566] [General]
Hi, 

So I trained a few parametric regression models. Through some scoring metrics, the coxph model with a piecewise equation had the best performance. 

My data is looking at subscribers/churners who started from August 2022 to October 9th 2022. The maximum duration in Months is 14 for censored subjects. 

I want to predict the additional months of survival for the censored subjects. I tried to use the coxph.predict_survival_function and pass in the duration column of the censored subjects. But I get this error:

![Screenshot 2023-10-17 at 12 03 58 PM](https://github.com/CamDavidsonPilon/lifelines/assets/46286577/f87df002-3a67-4a3a-9318-70a5d230e981)

Am i doing this right or is this an implementation issue regarding the coxph fitter?

**Comment by dlee07** [comment]
Hi @CamDavidsonPilon , 
just wanted an update on this question

**Reply by CamDavidsonPilon** [reply]
It hasn't been implemented for the piecewise model, likely because it's difficult.

**Reply by dlee07** [reply]
Hi @CamDavidsonPilon 

Thank you for the response. An unrelated question: Should the average predicted survival curve of an AFT Model be close to the kaplan meier curve? This has a concordance index of 0.78

<img width="587" alt="Screenshot 2023-11-13 at 4 23 32 PM" src="https://github.com/CamDavidsonPilon/lifelines/assets/46286577/64e42739-a142-4cc5-a721-29e9e0eb9169">

In addition, what is the benchmark ICI and E50 that I should be aiming for here? 

<img width="1012" alt="Screenshot 2023-11-13 at 4 24 10 PM" src="https://github.com/CamDavidsonPilon/lifelines/assets/46286577/1accddec-c3fe-42d5-8d05-ef8a5faa9969">
